packages/testing-helpers/index.js

```diff
@@ -16,5 +14,3 @@ export {
 
 export { litFixture, litFixtureSync } from './src/litFixture.js';
 export { stringFixture, stringFixtureSync } from './src/stringFixture.js';
-
-export { html, unsafeStatic };
```

Comment:
```js
import { html } from '@open-wc/testing';
```

is to my knowledge the only way to make sure that `html` matches with the `render` that testing expects...

e.g. it's the only way to be able to write a template fixture in lit 2 while my components are using lit 1 for example

Reply:
> is to my knowledge the only way to make sure that html matches with the render that fixture in testing expects...

But if your source code uses lit-html v1, or your storybook stories (which we reuse quite a bit for tests), that's not enough